tests/typechecker/return-value-type-mismatch.jakt

```diff
@@ -0,0 +1,3 @@
+function foo() -> u64 {
+        return "Foo"
```

Comment:
Four spaces for indentation please

Reply:
Whoops, my bad. Still got to fix my editor. Fixed. :^)